.github/workflows/cmake.yml

```diff
@@ -18,20 +18,16 @@ jobs:
           Ubuntu GCC No Opt,
           Ubuntu GCC Compat,
           Ubuntu GCC Compat No Opt,
-          Ubuntu GCC Compat,
           Ubuntu GCC ARM SF,
           Ubuntu GCC ARM SF No ACLE,
           Ubuntu GCC ARM SF No Opt,
           Ubuntu GCC ARM SF Compat,
-          Ubuntu GCC ARM SF Compat No Opt,
           Ubuntu GCC ARM HF,
           Ubuntu GCC ARM HF No NEON,
           Ubuntu GCC ARM HF No Opt,
           Ubuntu GCC ARM HF Compat,
-          Ubuntu GCC ARM HF Compat No Opt,
```

Comment:
I feel like we could rather do "Compat No Opt" to replace both "Compat" and "No Opt", removing two instead of one.
The two options do not interact much, if at all, so should be fine to test together.

Reply:
Same with the SF variant and AARCH64 variant.

Reply:
Done.